src/main/kotlin/no/nav/familie/ef/sak/oppgave/OppgaveService.kt

```diff
@@ -72,6 +72,20 @@ class OppgaveService(private val oppgaveClient: OppgaveClient,
         }
     }
 
+    private fun finnAktuellMappe(enhetsnummer: String?, oppgavetype: Oppgavetype): Long? {
+        if (enhetsnummer == "4489" && oppgavetype == Oppgavetype.GodkjenneVedtak) {
+            val mapper = finnMapper("4489")
+            val mappeIdForGodkjenneVedtak = mapper.find { it.navn.contains("EF Sak - 70 Godkjenne vedtak") }?.id?.toLong()
+            mappeIdForGodkjenneVedtak?.let {
+                logger.info("Legger oppgave i Godkjenne vedtak-mappe")
+            } ?: run {
+                logger.error("Fant ikke mappe for godkjenne vedtak: EF Sak - 70 Godkjenne vedtak")
+            }
```

Comment:
git run noe ekstar her eller kunne det like gjerne vært ?: logger.error("Fant ikke mappe for godkjenne vedtak: EF Sak - 70 Godkjenne vedtak")

Reply:
Ditt forslag utfører ikke kommandoen til logger.error, men vi justerer til egen metode med en `also` - så blir det ikke så iøenfallende i den reelle logikken :)